Created Docker Compose configuration file for project development.

dzenzerovic opened the pull request:
Contains the following services and their respective versions where specified:

- php-fpm:7.2
- nginx
- elasticsearch:7.12.1
- mariadb:10.6
- adminer
- tika:1.26
- php:8

All containers connect to a user-defined network named "project" and are able to communicate among themselves. The default root password for MariaDB is "example".

Commit:
Created Docker Compose configuration file for project development.

tic984 commented:
@dzenzerovic Why do we have php-fpm:7.3 and php:8 both in the project?

Also, why is adminer added? Please remove php7.2 and adminer.

Also add a file called Documentation.md, and describe how to start the containers and what are the default credentials there.

dzenzerovic commented:
> @dzenzerovic Why do we have php-fpm:7.3 and php:8 both in the project?

I apologize for the inconvenience; I misunderstood the task requirements and did not realize the PHP 8 container was redundant.

> Also, why is adminer added? Please remove php7.2 and adminer.

I added adminer because I was following the instructions related to setting up MariaDB using Docker Compose and was unaware it was not actually necessary, but rather given as an example. I will remove both containers, add php:8-fpm, and push these changes immediately.

> Also add a file called Documentation.md, and describe how to start the containers and what are the default credentials there.

I plan to have the documentation written and added by tomorrow afternoon, if not sooner. Do you have any additional guidelines regarding the documentation?

Commit:
Removed adminer and php containers; replaced php-fpm:7.2 with php:8-fpm
